### DISCUSSION TITLE
Best prompts and settings to test a trained model?

### DISCUSSION THREAD

**Discussion by broadweel** [TheLastBen/fast-stable-diffusion#1240] [Q&A]
Hello, I'm trying to train a face model and I really don't know what to use as prompt to test the differences between the ckpt steps. Is there any best formula to test the trained models, so that I can compare the changes between one checkpoint and another? What do you usually use to test the trained models?

If possible, tell me the prompts and settings used: prompt, negative prompts, sampling method, steps, width, height, cfg, seed, etc...

Thanks!

**Comment by TheLastBen** [comment]
hi, use lexica.art to inspire for good prompts, for the sampler, stick with Euler a
